lib/rust/data-structures/src/circular_vec.rs

```diff
@@ -0,0 +1,96 @@
+//! Definition of a circular vector, a vector with a constant max size, that if full, keeps its
+//! element in a loop.
+
+use std::collections::VecDeque;
+
+
+
+/// A vector with a constant max size, that if full, keeps its element in a loop.
```

Comment:
I don't understand this line. It does not `keep` elements, but rather drops the old ones when the capacity is full, doesn't it?